### PR TITLE
[TFLite] 16x8 quantization: fixes for SLICE, TRANSPOSE operators

### PR DESCRIPTION
**Pull request by wwwind**
This PR adds fixes for operators SLICE, TRANSPOSE to make them work for a model quantized in 16x8 quantization mode.
I haven't added versioning for these operators, because it is missing or inconsistent currently.
These changes have been done to make quantization/inference work on a real model.

### CODE REVIEW

File: tensorflow/lite/kernels/slice.cc [hunk 1]
```diff
@@ -214,6 +214,9 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
     case kTfLiteInt8:
       TF_LITE_SLICE(int8_t, kernel_type);
       break;
+    case kTfLiteInt16:
+      TF_LITE_SLICE(int16_t, kernel_type);
```
Comment (renjie-liu) [inline tensorflow/lite/kernels/slice.cc:218]: can you update the version?

if it's missing/inconsistent, I feel like we should fix it.

File: tensorflow/lite/toco/tflite/op_version.cc [hunk 2]
```diff
@@ -121,6 +121,7 @@ std::string GetMinimumRuntimeVersionForModel(const Model& model) {
           {{OperatorType::kTranspose, 1}, "1.6.0"},
           {{OperatorType::kTranspose, 2}, "1.14.0"},
           {{OperatorType::kTranspose, 3}, "1.15.0"},
+          {{OperatorType::kTranspose, 5}, kPendingReleaseOpVersion},
```
Comment (wwwind) [inline tensorflow/lite/toco/tflite/op_version.cc:124]: this place confused me - should I add version 4 with kPendingReleaseOpVersion ?

**Comment by wwwind** [comment]
Hi @renjie-liu I added versioning for 16x8 case to these operators. Please take a look. Thanks!

File: tensorflow/lite/toco/tflite/op_version.cc (hunk 2 above)
Comment (renjie-liu) [reply tensorflow/lite/toco/tflite/op_version.cc:124]: Seems it's "2.3.0"?

+jdduke@

**Review by renjie-liu — APPROVED** [review]

**Pull request merged**